Package.swift

```diff
@@ -0,0 +1,47 @@
+// swift-tools-version:5.3
+import PackageDescription
```

Comment:
wtf is swift doing here? does treesitter require rust, npm, C, and now swift?

Reply:
Not require, but support. These bindings are boilerplate that every parser gets so C, JS, Rust, Swift, Python projects can use tree-sitter, including parsers.

Reply:
Can all these things live in one place, instead of the top level? It's hard to tell what is generated and what isn't. That makes the project look way more overgrown than it is.

Reply:
You have a decision to make: do you want this to be a real parser that takes part in the tree-sitter ecosystem, or something internal that is only used for Neovim? (I personally think being considerate and standardize -- which upstream has started to do -- cuts both ways. )

Reply:
> Can all these things live in one place, instead of the top level?

Some, yes, but not all. It depends on the language.

> It's hard to tell what is generated and what isn't.

Look at `.gitattributes`.

Reply:
I'll revert this PR anyway since it massively broke stuff. Next PR should be more targeted and not add unrelated things in a single PR.

Reply:
The only unrelated thing is the modeline and that was for the tests.
It's fine to revert that part but you will also need to remove the file parsing.